src/components/Layout.js

```diff
@@ -52,17 +52,25 @@ const StyledBannerNotification = styled(BannerNotification)`
   text-align: center;
 `
 
-// TODO `Layout` renders twice on page load - why?
+function isDarkThemeUtil(isUseEffect, setThemeCb) {
```

Comment:
consider refactoring to arrow syntax to match codebase convention
ie: `const isDarkThemeUtil = (isUseEffect, setThemeCb) => { ... }`